packages/@tinacms/cli/src/next/commands/build-command/server/index.ts

```diff
@@ -0,0 +1,126 @@
+import { build, createServer as createViteServer, InlineConfig } from 'vite'
+import bodyParser from 'body-parser'
+import cors from 'cors'
+import { resolve as gqlResolve, Database } from '@tinacms/graphql'
+import { tinaTailwind } from '../tailwind'
+import { ConfigManager } from '../../../config-manager'
+import { createMediaRouter } from './routes'
+import { parseMediaFolder } from './models/media'
+import { transform as esbuildTransform } from 'esbuild'
+import { pipeline } from 'readable-stream'
+import { createServer } from 'net'
+import { ManyLevelHost } from 'many-level'
+import { MemoryLevel } from 'memory-level'
+
+export const createDBServer = () => {
```

Comment:
Looks like this functions is declared in 2-3 places. Maybe it should be unified into one place?